src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/TimeOutLicencesIntegrationTest.kt

```diff
@@ -61,28 +61,30 @@ class TimeOutLicencesIntegrationTest : IntegrationTestBase() {
     val jobExecutionDate = LocalDate.now()
     val isOnBankHolidayOrWeekEnd = workingDaysService.isNonWorkingDay(jobExecutionDate)
 
-    assertThat(timedOutLicences?.size).isEqualTo(if (isOnBankHolidayOrWeekEnd) 0 else 6)
+    val expected = if (isOnBankHolidayOrWeekEnd) {
+      listOf(
+        tuple(1L, LicenceStatus.IN_PROGRESS),
+        tuple(2L, LicenceStatus.IN_PROGRESS),
+        tuple(4L, LicenceStatus.IN_PROGRESS),
+        tuple(7L, LicenceStatus.IN_PROGRESS),
+        tuple(8L, LicenceStatus.IN_PROGRESS),
+        tuple(9L, LicenceStatus.IN_PROGRESS),
+      )
+    } else {
+      listOf(
+        tuple(1L, LicenceStatus.TIMED_OUT),
+        tuple(2L, LicenceStatus.TIMED_OUT),
+        tuple(4L, LicenceStatus.TIMED_OUT),
+        tuple(9L, LicenceStatus.TIMED_OUT),
+      )
+    }
+
+    assertThat(timedOutLicences?.size).isEqualTo(if (isOnBankHolidayOrWeekEnd) 0 else 4)
     assertThat(timedOutLicences)
       .extracting<Tuple> {
         tuple(it.licenceId, it.licenceStatus)
       }
-      .contains(
-        if (isOnBankHolidayOrWeekEnd) {
-          tuple(1L, LicenceStatus.IN_PROGRESS)
-          tuple(2L, LicenceStatus.IN_PROGRESS)
-          tuple(4L, LicenceStatus.IN_PROGRESS)
-          tuple(7L, LicenceStatus.IN_PROGRESS)
-          tuple(8L, LicenceStatus.IN_PROGRESS)
-          tuple(9L, LicenceStatus.IN_PROGRESS)
-        } else {
-          tuple(1L, LicenceStatus.TIMED_OUT)
-          tuple(2L, LicenceStatus.TIMED_OUT)
-          tuple(4L, LicenceStatus.TIMED_OUT)
-          tuple(7L, LicenceStatus.TIMED_OUT)
-          tuple(8L, LicenceStatus.TIMED_OUT)
-          tuple(9L, LicenceStatus.TIMED_OUT)
-        },
```

Comment:
this isn't valid syntax to build up a list of varargs - as it's inside a block it effectively only returned the final tuple so equated to:
`.contains(tuple(9L, LicenceStatus.TIMED_OUT))`

The missing commas was a clue - I've swapped this over to compare all elements of a list and chosen list based on whether it's a bank holiday or not

Reply:
The logic for if its a weekend or  bank holiday looks a bit suspect too

Reply:
I've removed the assertions if the tests are run during the weekend/bankholiday